scripts/update_copyright_date.sh

```diff
@@ -53,9 +53,29 @@ for i in `cat files2change`
 do
     echo $i
     cp $i $i.sed.bak
-    sed "s/Copyright (c) 2017-2019/Copyright (c) 2017-2020/" $i.sed.bak > $i
+    sed "s/Copyright (c) 2017-2020/Copyright (c) 2017-2021/" $i.sed.bak > $i
 done
 
+echo LICENSE
+cp LICENSE LICENSE.sed.bak
+sed "s/Copyright (c) 2017-2020/Copyright (c) 2017-2021/" LICENSE.sed.bak > LICENSE
+
+echo RELEASE
```

Comment:
I think these lines either needs some tweaks, or an initial manual intervention

Reply:
@kennyweiss thanks. I think I fixed everything. I shouldn't do a PR while I'm in a meeting. Sorry.